server/src/cmds/run/mod.rs

```diff
@@ -72,11 +63,10 @@ fn resolve_zone() -> Result<String, Error> {
             p = &p[1..];
         }
 
-        p = trim_zoneinfo(p);
-
-        if !p.starts_with('/') {
```

Comment:
This was handling the case where the zone is set right in the `TZ` environment variable, like `TZ=:America/Los_Angeles`. We need to keep supporting that. It's my recommended configuration in the Docker install instructions.